### DISCUSSION TITLE
Jest tests stopped working after the update?

### DISCUSSION THREAD

**Discussion by Refrezsh** [resend/react-email#1782] [Q&A] [answered]
After updating react-email to `v0.0.28`, all Jest tests stopped working, when trying to run a test, we get `Process finished with exit code 139 (interrupted by signal 11:SIGSEGV).` The problem occurs exactly at the moment of calling `await render()`. With that said, everything works, it also works if using Mocha as test runner. 

Jest starts working only if I reduce the version to `v0.0.22`, in this version render is still synchronous. Has anyone encountered such a problem ?

**Comment by gabrielmfern** [comment, marked as answer]
Internally, we import `react-dom` dynamically. IIRC, Jest doesn't support these unless you set it up

https://jestjs.io/docs/next/ecmascript-modules

**Reply by Refrezsh** [reply]
Thank you so much.
